src/ngx_http_lua_resolver.c

```diff
@@ -0,0 +1,367 @@
+
+#ifndef DDEBUG
+#define DDEBUG 0
+#endif
+#include "ddebug.h"
+
+
+#include "ngx_http_lua_util.h"
+#include "ngx_http_lua_contentby.h"
+
+
+typedef struct ngx_http_lua_resolver_ctx_s {
+    ngx_http_request_t              *request;
+    u_char                          *buf;
+    size_t                          *buf_size;
+    ngx_http_lua_co_ctx_t           *curr_co_ctx;
+    ngx_resolver_ctx_t              *resolver_ctx;
+    ngx_int_t                        exit_code;
+    unsigned                         ipv4:1;
+    unsigned                         ipv6:1;
+} ngx_http_lua_resolver_ctx_t;
+
+
+static void ngx_http_lua_resolve_handler(ngx_resolver_ctx_t *rctx);
+static void ngx_http_lua_resolve_cleanup(void *data);
+static void ngx_http_lua_resolve_empty_handler(ngx_resolver_ctx_t *ctx);
+static ngx_int_t ngx_http_lua_resolve_resume(ngx_http_request_t *r);
+static int ngx_http_lua_resolve_retval(ngx_http_lua_resolver_ctx_t *ctx,
+                                       lua_State *L);
```

Comment:
Bad indentation. See the guide for details:

https://openresty.org/en/c-coding-style-guide.html

Also, make sure you run the `ngx-releng` tool mentioned in the guide as well. That tool should catch this.